proxies/live/apiproxy/proxies/default.xml

```diff
@@ -69,6 +69,15 @@
   <RouteRule name="communications-manager-target">
     <TargetEndpoint>communications-manager-target</TargetEndpoint>
   </RouteRule>
+
+  <FaultRules>
+   <FaultRule name="missing_parameter">
+      <Step>
+        <Name>RaiseFault.400BadRequest</Name>
```

Comment:
Probably needs a different error, we can leave it as it is now as our focus is on sandbox.